modules/collecting-gitops-debugging-data.adoc

```diff
@@ -36,7 +43,8 @@ The `must-gather` tool creates a new directory that starts with `./must-gather.l
 +
 [source,terminal]
 ----
-$ tar -cvaf must-gather.tar.gz must-gather.local.4157245944708210399
+$ tar -cvaf must-gather.tar.gz must-gather.local.4157245944708210399 <1>
 ----
+<1> Replace `must-gather-local.4157245944708210399/` with the actual directory name.
```

Comment:
```suggestion
<1> Replace `must-gather-local.4157245944708210399` with the actual directory name.
```
Should the slash be removed here?

Reply:
@lahinson Thank you so much! Addressed in both the PRs.